test/models/city_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class CityTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
Well done @atila-tokyo now we just need to add some unit tests for Country, City and User if you feel in the mood.